packages/@react-types/image/src/index.d.ts

```diff
@@ -13,11 +13,29 @@
 import {DOMProps, StyleProps} from '@react-types/shared';
 
 export interface ImageProps {
-  loaded?: boolean, // not real, just makes ts happy for now
-  isPlaceholder?: boolean, // same thing
+  /**
+   * Whether the Image should be displayed with a loaded style.
+   */
+  loaded?: boolean,
```

Comment:
I know this is out of this PR's scope, but shouldn't this be `isLoaded` following the `isSomething` convention for booleans?